### PR TITLE
Add version option in command line

### LINKED ISSUE #119: Add `-v` and `--version` options
For reproducibility, it's often useful to output the version of the tool we use.

It might be pertinent for `buildH` to provide its version as well. For instance:
```
$ buildH -v
1.3.1
$ buildH --version
1.3.1
```

### PR DESCRIPTION
**Pull request by pierrepo**
Fix #119

### CODE REVIEW

**Comment by pierrepo** [comment]
@HubLot and @patrickfuchs I couldn't get the version from ` __init__.py` so I had to duplicate it in `UI.py`... This is not optimal. Do you have better alternative?

**Comment by patrickfuchs** [comment]
I looked a bit at this, not easy at all...! Maybe solution 6 described in https://packaging.python.org/guides/single-sourcing-package-version/ ?

**Comment by pierrepo** [comment]
Humm not sure since I'm already in the submodule.

File: buildh/UI.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
 from . import writers
 from . import utils
 
+__version__ = "1.3.1"
+
```
Comment (HubLot) [inline buildh/UI.py:22]: `from . import __version__` should to the trick

Comment (pierrepo) [reply buildh/UI.py]: Thanks @HubLot 
It works!

**Comment by pierrepo** [comment]
Thanks to @HubLot suggestion, it looks much better now.
@patrickfuchs and @HubLot could you have a close look that I did not break anything?

**Review by patrickfuchs — APPROVED** [review]
To me it's fine 😄

**Review by HubLot — APPROVED** [review]

**Comment by pierrepo** [comment]
@patrickfuchs if you agree with the PR, could you please merge?

**Pull request merged**